zebra-chain/src/transaction.rs

```diff
@@ -130,4 +130,12 @@ impl Transaction {
             Transaction::V4 { expiry_height, .. } => Some(*expiry_height),
         }
     }
+
+    /// Does this transaction contain any coinbase inputs?
```

Comment:
```suggestion
    /// Returns `true` if transaction does not contain any coinbase inputs.
```

Reply:
@yaahc why "does not contain" ?

`any(... TransparentInput::Coinbase { .. } => true ...)` should return true when the transaction contains one or more coinbase inputs.

Reply:
oh you're right, fixed